### PR TITLE
New Home Page ++

### PR DESCRIPTION
**Pull request by gib**
new-home

- New home page design from [StudioFluid](http://www.studiofluid.com/)
- Some prep styles / author page for my blog post (coming in a PR to [aptible/blog](https://github.com/aptible/aptible-blog)
- Quick and dirty product feature page partial - will have a sample use in a PR to [aptible-pages](https://github.com/aptible/aptible-pages) (@chasballew)
- Replaces Company with FAQ to the top nav (@samyount)

### CODE REVIEW

**Comment by fancyremarker** [comment]
:clap: :clap: :clap:

File: src/assets/javascript/images.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+$ ->
+  if window.devicePixelRatio == 2
+    $("img[src$='product-stack.png']").attr('src', '/assets/images/product-stack-2x.png')
```
Comment (gib) [inline src/assets/javascript/images.coffee]: This is a bummer. The SVG output from Illustrator was complicated enough to require masks. These didn't work in Safari. I expect I could spend some time and figure this out, but in the interest of time, I went with this.

File: src/assets/stylesheets/_home.scss [hunk 2]
```diff
@@ -225,31 +474,6 @@ header.primary-header {
...
-
-    }
-  }
-
-  .tools-description {
-    text-align: center;
-    margin: 40px 0 0;
-  }
-}
-
```
Comment (gib) [inline src/assets/stylesheets/_home.scss]: Could clean up more styles here. Tabling for the next big round that may included ditching Assemble for Middleman.

File: src/partials/header.hbs [hunk 3]
```diff
@@ -1,44 +1,19 @@
-<header class="primary-header" {{#is dirname 'dist/technology'}}style="opacity:0"{{/is}}>
```
Comment (gib) [inline src/partials/header.hbs]: Hiding the nav initially via scoped CSS rather than here inline.

**Comment by gib** [comment]
Merging, but comments are open / welcome!

File: src/assets/javascript/images.coffee (hunk 1 above)
Comment (chasballew) [inline src/assets/javascript/images.coffee]: :runner:

**Comment by chasballew** [comment]
This look excellent, really strong. Thanks!

**Comment by shahkader** [comment]
Well done!

**Pull request merged**